source/deployment/sso-saml-ldapsync.rst

```diff
@@ -56,9 +56,12 @@ When enabled, SAML synchronization with AD/LDAP occurs in phases:
 Override SAML Data with AD/LDAP Data
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  
-Alternatively, you can choose to override the SAML bind data with the AD/LDAP information. This will help ensure that new users are not created when the email address changes for a user. 
+Alternatively, you can choose to override the SAML bind data with the AD/LDAP information. This will process will override SAML email address with AD/LDAP email address data or SAML Id Attribute with AD/LDAP Id Attribute if configured. For more infomation binding user with the SAML ID Attribute, please refer to this `documentation <https://docs.mattermost.com/deployment/sso-saml-okta.html#bind-authentication-to-id-attribute-instead-of-email>`_.  
```

Comment:
Propose

1. `Alternatively, you can choose to override SAML bind data with AD/LDAP information. This process overrides SAML email address with AD/LDAP email address data or SAML Id Attribute with AD/LDAP Id Attribute if configured.`

2. On the above sentence, for `...if configured`, where would the admin do the configuration?
3. The first sentence talks about overriding SAML data, but the second talks about SAML ID attribute. It's not immediately obvious how the two are related. It's more clear after the second paragraph. Hence, wondering if second sentence would be moved at the end, or after, the second paragraph?
4. For the link, propose linking to https://docs.mattermost.com/deployment/saml-faq.html#bind-authentication-to-id-attribute-instead-of-email instead of the OKTA-specific page (that way it'll be more clear it's not OKTA-specific)
5. For https://docs.mattermost.com/deployment/saml-faq.html#bind-authentication-to-id-attribute-instead-of-email, propose renaming `saml-faq.rst` filename to `sso-saml-faq.rst` so that it's consistent with the SAML page naming convention.

Reply:
Updated based on feedback and made separate paragraphs for additional readability.